tsl/test/expected/bgw_policy.out

```diff
@@ -621,3 +621,101 @@ GROUP BY proc_name;
  policy_retention |     2
 (2 rows)
 
+-- test that removing the strict specifier has no effect on expected behavior
```

Comment:
none of the following tests remove a strict specifier

Reply:
What I meant here was to test that “the function still behaves as strict even though STRICT was removed from the function definition”